ERROR - 2024-05-25 04:43:53 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'pos', 3306, '', 0)
#1 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pos\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pos\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pos\app\Controllers\Unidades.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pos\system\CodeIgniter.php(933): App\Controllers\Unidades->index()
#8 C:\xampp\htdocs\pos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
#9 C:\xampp\htdocs\pos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pos\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\pos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\pos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pos\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pos\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pos\app\Controllers\Unidades.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pos\system\CodeIgniter.php(933): App\Controllers\Unidades->index()
#7 C:\xampp\htdocs\pos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
#8 C:\xampp\htdocs\pos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pos\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\pos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\pos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-05-25 04:43:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: GET, Route: unidades]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `unidades`
WHERE `activo` = :activo:', [...], false)
 3 SYSTEMPATH\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Unidades.php(20): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Unidades->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-05-25 04:44:24 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'pos', 3306, '', 0)
#1 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pos\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pos\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pos\app\Controllers\Unidades.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pos\system\CodeIgniter.php(933): App\Controllers\Unidades->index()
#8 C:\xampp\htdocs\pos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
#9 C:\xampp\htdocs\pos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pos\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\pos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\pos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pos\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pos\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pos\app\Controllers\Unidades.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pos\system\CodeIgniter.php(933): App\Controllers\Unidades->index()
#7 C:\xampp\htdocs\pos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
#8 C:\xampp\htdocs\pos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pos\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\pos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\pos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-05-25 04:44:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: GET, Route: unidades]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `unidades`
WHERE `activo` = :activo:', [...], false)
 3 SYSTEMPATH\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Unidades.php(20): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Unidades->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-05-25 04:44:42 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'pos', 3306, '', 0)
#1 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pos\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pos\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pos\app\Controllers\Unidades.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pos\system\CodeIgniter.php(933): App\Controllers\Unidades->index()
#8 C:\xampp\htdocs\pos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
#9 C:\xampp\htdocs\pos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pos\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\pos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\pos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pos\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pos\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pos\app\Controllers\Unidades.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pos\system\CodeIgniter.php(933): App\Controllers\Unidades->index()
#7 C:\xampp\htdocs\pos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
#8 C:\xampp\htdocs\pos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pos\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\pos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\pos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-05-25 04:44:42 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: GET, Route: unidades]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `unidades`
WHERE `activo` = :activo:', [...], false)
 3 SYSTEMPATH\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Unidades.php(20): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Unidades->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Unidades))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-05-25 04:44:59 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'pos', 3306, '', 0)
#1 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pos\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pos\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pos\app\Controllers\Categorias.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pos\system\CodeIgniter.php(933): App\Controllers\Categorias->index()
#8 C:\xampp\htdocs\pos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp\htdocs\pos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pos\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\pos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\pos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pos\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pos\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pos\app\Controllers\Categorias.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pos\system\CodeIgniter.php(933): App\Controllers\Categorias->index()
#7 C:\xampp\htdocs\pos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp\htdocs\pos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pos\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\pos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\pos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-05-25 04:44:59 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: GET, Route: categorias]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `activo` = :activo:', [...], false)
 3 SYSTEMPATH\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(20): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-05-25 04:45:25 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'pos', 3306, '', 0)
#1 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pos\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pos\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pos\app\Controllers\Categorias.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pos\system\CodeIgniter.php(933): App\Controllers\Categorias->index()
#8 C:\xampp\htdocs\pos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp\htdocs\pos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pos\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\pos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\pos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pos\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pos\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pos\app\Controllers\Categorias.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pos\system\CodeIgniter.php(933): App\Controllers\Categorias->index()
#7 C:\xampp\htdocs\pos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp\htdocs\pos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pos\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\pos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\pos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-05-25 04:45:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: GET, Route: categorias]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `activo` = :activo:', [...], false)
 3 SYSTEMPATH\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(20): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-05-25 04:45:26 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'pos', 3306, '', 0)
#1 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pos\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pos\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pos\app\Controllers\Categorias.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pos\system\CodeIgniter.php(933): App\Controllers\Categorias->index()
#8 C:\xampp\htdocs\pos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp\htdocs\pos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pos\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\pos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\pos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pos\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pos\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pos\app\Controllers\Categorias.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pos\system\CodeIgniter.php(933): App\Controllers\Categorias->index()
#7 C:\xampp\htdocs\pos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp\htdocs\pos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pos\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\pos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\pos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-05-25 04:45:26 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: GET, Route: categorias]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `activo` = :activo:', [...], false)
 3 SYSTEMPATH\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(20): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-05-25 04:45:56 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'pos', 3306, '', 0)
#1 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pos\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pos\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pos\app\Controllers\Categorias.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pos\system\CodeIgniter.php(933): App\Controllers\Categorias->index()
#8 C:\xampp\htdocs\pos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp\htdocs\pos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pos\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\pos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\pos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\pos\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pos\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pos\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pos\system\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pos\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pos\app\Controllers\Categorias.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pos\system\CodeIgniter.php(933): App\Controllers\Categorias->index()
#7 C:\xampp\htdocs\pos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp\htdocs\pos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pos\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\pos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\pos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-05-25 04:45:56 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: GET, Route: categorias]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `activo` = :activo:', [...], false)
 3 SYSTEMPATH\Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(20): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
